CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+- Adds the `--uri` flag to pass a MongoDB server connection string allowing for initial user addition during project creation when a host server is being used.
```

Comment:
Probably not the only URI we'll ever care about, so I suggest `mongodb-uri`.

Reply:
Changed.